gems/sorbet-runtime/lib/types/private/abstract/declare.rb

```diff
@@ -9,6 +9,9 @@ def self.declare_abstract(mod, type:)
     if AbstractUtils.abstract_module?(mod)
       raise "#{mod.name} is already declared as abstract"
     end
+    if T::Private::Final.is_final?(mod)
```

Comment:
What do you think about making this be named similarly to the `abstract_module?` like `final_class?` or `final?`

specifically the `is_` is redundant with the `?`